airflow/models/trigger.py

```diff
@@ -212,12 +213,14 @@ def assign_unassigned(cls, triggerer_id, capacity, session: Session = NEW_SESSIO
 
         if capacity <= 0:
             return
-
+        # we multiply heartrate by a grace_multiplier to give the triggerer
+        # a chance to heartbeat before we consider it dead
+        health_check_threshold = heartrate * 2.1
```

Comment:
> Airflow tries to only run triggers in one place at once, and maintains a heartbeat to all triggerers that are currently running. If a triggerer dies, or becomes partitioned from the network where Airflow’s database is running, Airflow will automatically re-schedule triggers that were on that host to run elsewhere (after waiting 30 seconds for the machine to re-appear).

https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/deferring.html#high-availability

I feel this could also be updated to reference the new configuration instead of 30 seconds. As a personal opinion multiplier can be an integer 2 or 3 than 2.1 specifically and also maybe a named constant.

Reply:
Actually we use this value as default `grace_multiplier` in the job `is_alive` method, IMHO it's better to use the same one here to be coherent with that method:
https://github.com/apache/airflow/blob/546588a9dd02860e8bba2e47b8d87b18927c6186/airflow/jobs/job.py#L123-L140

Reply:
Got it, thanks.